.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
```

Comment:
A single line `npx --no-install lint-staged` would do. Use `--no-install` as we have already installed it locally. What's the code above it for?